File: src/Domain/Entities/Comment.cs
using System;
using SharedPhotoAlbum.Domain.Common;

namespace SharedPhotoAlbum.Domain.Entities
{
    public class Comment : AuditableEntity
    {
        public Guid Id { get; set; }

        public string Text { get; set; }
        
        public int Likes { get; set; }
        
        public Guid PostId { get; set; }
        
        public Post Post { get; set; }
    }
}

File: src/WebUI/ClientApp/src/components/models/Comment.ts
export class Comment {
    id?: string
    text?: string | undefined
    likes?: number
    postId?: string

    constructor(id: string, text: string, likes: number, postId: string) {
        this.id = id
        this.text = text
        this.likes = likes
        this.postId = postId
    }
}
